core/kotlinx-coroutines-core/test/internal/LockFreeTaskQueueStressTest.kt

```diff
@@ -34,7 +34,7 @@ class LockFreeMPSCQueueStressTest : TestBase() {
         val threads = mutableListOf<Thread>()
         threads += thread(name = "Pacer", start = false) {
             while (done.value == 0) {
-                queue.value = LockFreeMPSCQueue()
+                queue.value = LockFreeTaskQueue(false)
```

Comment:
uhm, but single-consumer pattern is still tested.
Please parametrize this test properly

Reply:
Done